Microsoft.Toolkit.Win32/Microsoft.Toolkit.Wpf.UI.Controls/InkToolbar/InkToolbar.cs

```diff
@@ -0,0 +1,287 @@
+
+        /// <summary>
+        /// Gets <see cref="Windows.UI.Xaml.Controls.InkToolbar.IsStencilButtonCheckedProperty"/>
+        /// </summary>
+        public static DependencyProperty IsStencilButtonCheckedProperty { get; } = DependencyProperty.Register(nameof(IsStencilButtonChecked), typeof(bool), typeof(InkToolbar));
+
+        /// <summary>
+        /// Gets <see cref="Windows.UI.Xaml.Controls.InkToolbar.OrientationProperty"/>
+        /// </summary>
+        public static DependencyProperty OrientationProperty { get; } = DependencyProperty.Register(nameof(Orientation), typeof(Orientation), typeof(InkToolbar));
+
+        /// <summary>
+        /// <see cref="Windows.UI.Xaml.Controls.InkToolbar.GetToolButton"/>
+        /// </summary>
+        /// <returns>WindowsXamlHostBaseExt</returns>
+        public WindowsXamlHostBaseExt GetToolButton(InkToolbarTool tool) => UwpControl.GetToolButton((Windows.UI.Xaml.Controls.InkToolbarTool)(int)tool).GetWrapper();
+
+        /// <summary>
+        /// <see cref="Windows.UI.Xaml.Controls.InkToolbar.GetToggleButton"/>
+        /// </summary>
+        /// <returns>InkToolbarToggleButton</returns>
+        public InkToolbarToggleButton GetToggleButton(InkToolbarToggle tool) => (InkToolbarToggleButton)UwpControl.GetToggleButton((Windows.UI.Xaml.Controls.InkToolbarToggle)(int)tool).GetWrapper();
+
+        /// <summary>
+        /// <see cref="Windows.UI.Xaml.Controls.InkToolbar.GetMenuButton"/>
+        /// </summary>
+        /// <returns>InkToolbarMenuButton</returns>
+        public InkToolbarMenuButton GetMenuButton(InkToolbarMenuKind menu) => (InkToolbarMenuButton)UwpControl.GetMenuButton((Windows.UI.Xaml.Controls.InkToolbarMenuKind)(int)menu).GetWrapper();
+
+        /// <summary>
+        /// Gets or sets <see cref="Windows.UI.Xaml.Controls.InkToolbar.TargetInkCanvas"/>
+        /// </summary>
+        public InkCanvas TargetInkCanvas
+        {
+            get => (InkCanvas)GetValue(TargetInkCanvasProperty);
+            set => SetValue(TargetInkCanvasProperty, value);
+        }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether <see cref="Windows.UI.Xaml.Controls.InkToolbar.IsRulerButtonChecked"/>
+        /// </summary>
+        public bool IsRulerButtonChecked
+        {
+            get => (bool)GetValue(IsRulerButtonCheckedProperty);
+            set => SetValue(IsRulerButtonCheckedProperty, value);
+        }
+
+        /// <summary>
+        /// Gets or sets <see cref="Windows.UI.Xaml.Controls.InkToolbar.InitialControls"/>
+        /// </summary>
+        public InkToolbarInitialControls InitialControls
+        {
+            get => (InkToolbarInitialControls)GetValue(InitialControlsProperty);
+            set => SetValue(InitialControlsProperty, value);
+        }
+
+        /// <summary>
+        /// Gets or sets <see cref="Windows.UI.Xaml.Controls.InkToolbar.ActiveTool"/>
+        /// </summary>
+        public WindowsXamlHostBaseExt ActiveTool
```

Comment:
Does this need to be `WindowsXamlHostBaseExt` or can it be `WindowsXamlHostBase`?